platform/ios/PolyPodApp/PolyPod/SettingsView.swift

```diff
@@ -91,6 +92,33 @@ private struct MainSection: View {
                 EdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 0)
             )
             
+
+            Section(header: SettingsHeader("settings_sec_section")) {
+                SettingsToggleButton(
+                    label: "settings_auth",
+                    isToggle: $isToggle,
+                    onChange: { status in
```

Comment:
This is where you're using the Binding that's extended above, right?

Reply:
yep exactly :)